### DISCUSSION TITLE
Hope avalonia has a similar project~~

### DISCUSSION THREAD

**Discussion by CeSun** [Dreamescaper/BlazorBindings.Maui#62] [General]
this project is awesome, It's a pity to be dragged down by MAUI.MAUI has too many bugs.

Maybe it would be better if avalonia had a similar framework.

**Comment by Dreamescaper** [comment]
Yeah, I was thinking to build `BlazorBindings.Avalonia` or `BlazorBindings.Uno` POC at some point. 
As for now, I've decided to focus on MAUI, stabilize the API, etc. Since both MAUI and Avalonia are XAML-focused frameworks, I suppose challenges will be similar.
